tests/integration_tests/data_steward/tools/recreate_person_test.py

```diff
@@ -52,22 +50,32 @@ def setUp(self):
 
         self.fq_person = f'{self.project_id}.{self.dataset_id}.person'
         self.fq_person_ext = f'{self.project_id}.{self.dataset_id}.person_ext'
-        self.table_ids = [self.fq_person, self.fq_person_ext]
+        self.table_ids = [self.fq_person_ext, self.fq_person]
 
-        self.person = Table.from_string(self.fq_person)
-        self.person_ext = Table.from_string(self.fq_person_ext)
+        self.person = gbq.Table.from_string(self.fq_person)
+        self.person_ext = gbq.Table.from_string(self.fq_person_ext)
 
         self.client = bq.get_client(self.project_id)
-        for table in self.table_ids:
-            self.client.delete_table(table, not_found_ok=True)
-        bq.create_tables(self.client, self.project_id, self.table_ids)
+        self.tearDown()
```

Comment:
Can you explain why this `tearDown()` function is here?

Reply:
I used it to remove any existing person/person_ext tables that could interfere with the tests. Removing them beforehand ensures the tables can be created and populated as expected for the test.

Reply:
That makes sense, thank you!